src/limits.rs

```diff
@@ -0,0 +1,2 @@
+pub const MAX_CUMULATIVE_HEAP_SIZE: usize = 200 * 1024 * 1024;
+pub const MAX_ATOM_SIZE: usize = 1024 * 1024;
```

Comment:
I think these limits should be significantly higher than we would expect any normal program to need, does this seem like reasonable limits?

Reply:
Don't use constants, make them parameters.